fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -424,6 +424,18 @@ protected FunctionCallExpr(FunctionCallExpr other) {
         this.isTableFnCall = other.isTableFnCall;
     }
 
+    public String parseJsonValueModifyDataType() {
```

Comment:
This function also need to be implemented in Nereids planner

Reply:
I'll implement it